control/tests/sisotool_test.py

```diff
@@ -59,7 +59,7 @@ def test_sisotool(self):
         assert_array_almost_equal(ax_mag.lines[0].get_data()[1][10:20],bode_mag_moved)
 
         # Check if the step response has changed
-        step_response_moved = np.array([[ 0.,          0.02458187,  0.16529784 , 0.46602716 , 0.91012035 , 1.43364313, 1.93996334 , 2.3190105  , 2.47041552 , 2.32724853] ])
+        step_response_moved = np.array([0.    , 0.0239, 0.161 , 0.4547, 0.8903, 1.407 , 1.9121, 2.2989, 2.4686, 2.353 ])
```

Comment:
Same question as above.  Just want to make sure I understand the need for the change.